backend/inspirehep/migrator/tasks.py

```diff
@@ -200,6 +201,20 @@ def migrate_recids_from_mirror(
     return str(chord_task.id)
 
 
+@task_failure.connect
+def log_error(
+    task_id=None, exception=None, args=None, kwargs=None, traceback=None, einfo=None
+):
+    LOGGER.error(
+        "One of the tasks in chord just failed.",
+        task_id=task_id,
+        exception=exception,
+        task_args=args,
+        task_kwargs=kwargs,
+        traceback=traceback,
+    )
```

Comment:
This hooks into the signal far all tasks, so the message should be more generic, like `Celery task failed`, and it should be moved to the `logger.config` module with the other celery logging hooks.

Reply:
better to set `exc_info=(type(exception), exception, traceback)` so it gets interpreted correctly.